packages/taco/integration-test/encrypt-decrypt.test.ts

```diff
@@ -6,6 +6,9 @@ import {
   USER_ADDRESS_PARAM_DEFAULT,
 } from '@nucypher/taco-auth';
 import { ethers } from 'ethers';
+import { createPublicClient, http } from 'viem';
```

Comment:
If strong types was used below instead of `any`:
```suggestion
import { Account, createPublicClient, http, PublicClient } from 'viem';
```